client/src/pinboard.tsx

```diff
@@ -146,8 +149,8 @@ export const Pinboard = ({
         <HeadingPanel
```

Comment:
to avoid confusion with new `Panel` component
```suggestion
        <Header
```
or
```suggestion
        <PanelHeading
```

Reply:
will do this in #82

Reply:
resolved in #82 (renamed to navigation to align with the design name)